onnx/utils.py

```diff
@@ -148,6 +149,7 @@ def extract_model(
         output_path (string): The path to save the extracted ONNX model.
         input_names (list of string): The names of the input tensors that to be extracted.
         output_names (list of string): The names of the output tensors that to be extracted.
+        check_model (bool): The names of the output tensors that to be extracted.
```

Comment:
Typo: Update comment. "Whether to run model checker on the extracted model"

Reply:
Fixed. Thanks!